src/uct/ib/base/ib_iface.h

```diff
@@ -48,6 +48,13 @@ typedef enum {
     UCT_IB_DIR_NUM
 } uct_ib_dir_t;
 
+typedef enum {
+    UCT_IB_QPT_RC = IBV_QPT_RC,
+    UCT_IB_QPT_UD = IBV_QPT_UD,
```

Comment:
need to define QPT for DC only , this enum can be nameless and pass around qp_type as int